### DISCUSSION TITLE
Augment Partners: Kedro Cheat Sheet

### DISCUSSION THREAD

**Discussion by yetudada** [kedro-org/kedro#1054] [Idea]
We spotted this very handy cheat sheet by Augment Partners on LinkedIn: https://www.linkedin.com/posts/augment-partners_kedro-cheat-sheet-by-augment-activity-6858927624631283712-Ivqk/ 

Find the cheat sheet: 
[Augment_Partners_Kedro_Cheat_Sheet.pdf](https://github.com/quantumblacklabs/kedro/files/7569424/Augment_Partners_Kedro_Cheat_Sheet.pdf)

**Comment by meaningfromdata** [comment]
This is a helpful cheat sheet!  Thanks for posting it.
